app/services/appointment_fetcher.rb

```diff
@@ -0,0 +1,16 @@
+class AppointmentFetcher
+  def call(work_order_reference:)
+    @work_order_reference = work_order_reference
+
+    appointments = available_appointments
+    appointments.map { |appointment| AppointmentPresenter.new(appointment) }
+  end
+
+  private
+
+  def available_appointments
+    @available_appointments ||= HackneyApi.new.list_available_appointments(
```

Comment:
Since `available_appointments` is only called once inside this class, what do we get from memoizing it?